i18n/ru.yml

```diff
@@ -0,0 +1,160 @@
+            exampleurl: http://example.com
+    traffic:
+        header: Трафик 
+        table:
+            url: URL
+            visits: Посетители 
+    users:
+        header: Пользователи
+        table:
+            username: Имя пользователя 
+            password: Пароль 
+            actions: Действия 
+        actions:
+            update: Обновление
+            delete: Удалить
+        form:
+            new: Новый Пользователь 
+            username: Имя пользователя 
+            password: Пароль 
+            admin: Администратор 
+            user: Стандартный пользователь 
+            create: Создать 
+    videos:
+        header: Видео
+        table:
+            title: Название 
+            author: Автор
+            type: Тип
+            uri: URI
+            description: Описание
+            uploaded: Дата Публикации
+            modified: Дата Коррекция 
+            actions: Действия 
+            selector:
+                youtube: YouTube
+                vimeo: Vimeo
+        actions:
+            edit: Редактировать 
+            delete: Удалить 
+            create: Создать 
+        helpbeg: URI видео - Идентификатор в конце URL видео. Например, если вы хотели разместить,
+        helpend: то URI бы было
+blog:
+    header: Блог
+home:
+    title: Добро пожаловать в WULF
+    learnmore: Узнать больше 
+    news: Новости
+    readmore: Прочитать больше 
+    videos: Видео 
+    seemore: Смотреть больше 
+    blog: Блог
+    visitblog: Идти в блог
+header:
+    description: Официальный сайт Рабочего Единого Лебого Фронта 
+    author: WULF
+    title: WULF
+notfound:
+    header: 404 Not Found
+    content: Страница не получилась. Смотрите ссылки на верх, чтобы найти то, что вы ищете. 
+resource:
+    header: Ресурсы 
+    meta: Представлен %1 в %2
+    visit: Смотреть ресурс
+videos:
+    header: Видео
+types:
+    video: видео
+    resource: ресурс
+    news: новости 
+    article: статья
+about:
+    header: О Нам
+    content:Рабочий Единый Левый Фронт - Организация, главная цель которой то, что люди знают, что они угнетены и  были подозрительными деятельностей правительства, и чтобы политики были ответственными за них. Мы хотим, чтобы вы знали, что эти политики не сохраняют рабочий класс, ни освободят нас от власти другой стороны. Мы все видили недостаток изменении от администрации к администрации в качестве доказательства. Это недостаток из-за того, что политики врут постоянно вам о своём сострадании за вас, чтобы они обеспечили ваш голос. Цикл повторяется постоянно. Мы провозглашаем, что все решения в обществе должны быть в основе потребностей народы, вместо того, что генерирует прибыль. Мы провозглашаем, что все решения должны быть в руках народы и путем прямой демократии, и должны быть также децентрализованной, чтобы устранить все проблемы эффективнее. Этот процесс создаёт истинное освобождение всех народов от угнетения, и ни от такой-то власти, которая заботится только собой. Вся власть народу.    
```

Comment:
I'm thinking this is the source of the build error. I'll fix it in a new commit. https://travis-ci.org/WULF-USA/Official-Website/builds/198971305#L674